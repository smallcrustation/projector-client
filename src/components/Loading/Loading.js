import React from 'react'

import loading from './loading.gif'

export default class Loading extends React.Component{
  render(){
    return <img src={loading} alt='Loading...'></img>
  }
}