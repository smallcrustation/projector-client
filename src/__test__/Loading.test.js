import React from 'react'
import ReactDOM from 'react-dom'

import Loading from '../components/Loading/Loading'
import renderer from 'react-test-renderer'

describe('App Testing', () => {
  it('Renders without crashing', () => {
    const div = document.createElement('div')

    ReactDOM.render(<Loading />, div)

    ReactDOM.unmountComponentAtNode(div)
  })

  it('Renders with no crashing and matches snapshot', () => {
    const tree = renderer.create(<Loading />).toJSON()

    expect(tree).toMatchSnapshot()
  })
})
