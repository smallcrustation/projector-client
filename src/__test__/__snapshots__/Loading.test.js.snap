// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Testing Renders with no crashing and matches snapshot 1`] = `
<img
  alt="Loading..."
  src="loading.gif"
  style={
    Object {
      "display": "block",
      "marginLeft": "auto",
      "marginRight": "auto",
    }
  }
/>
`;
